vtex/faststore
# Feat: Adds `Skeleton` Component

## What's the purpose of this pull request?

- [x] Refactor Skeleton component and add it to @faststore/components
- [x] Adds Skeleton styles to @faststore/ui
- [x] Adds Skeleton doc page

How to test it?

run `yarn workspace site dev` and check the `Skeleton` component  doc page or test it using the doc preview link: [Skeleton Component](https://faststore-site-git-feat-skeleton-component-faststore.vercel.app/components/atoms/skeleton)

### Starters Deploy Preview

Feat: Replaces Skeleton FSUI #361: https://github.com/vtex-sites/nextjs.store/pull/361
